packages/vuetify/tsconfig.json

```diff
@@ -6,7 +6,13 @@
   "compilerOptions": {
     "baseUrl": ".",
     "paths": {
-      "@/*": ["src/*"]
+      "@/*": [
+        "src/*"
+      ],
+      "types": [
+        "jest",
+        "node"
+      ],
```

Comment:
Dunno how to handle this yet, but there should be some way - using Jest and Cypress in the same codebase is not something new. Previously I've kept all my cypress specs in a single directory, and had a `tsconfig.json` in that directory, which specified different types. I'm not sure how to handle things here, since ideally we want to keep the tests alongside the components. 🤔

Reply:
Actually, I think we can do it like this:

```
/// <reference types="cypress" />
```

On a per file basis.